Assets/Common/Scripts/DataTable/TableBase.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+
+
+public struct ObjectData
```

Comment:
ObjectData가 TableBase.cs에 같이 있는 게 어색합니다.
TableBase에 넣어보거나 Data 폴더에 넣어보는 게 어떨까요?
또한 데이터 저장용으로 쓰이는 struct인데 메모리가 커질 것 같다면 record를 써보는 것도 좋을 것 같습니다.
[관련 레퍼런스](https://daekyoulibrary.tistory.com/entry/C-%EB%A0%88%EC%BD%94%EB%93%9CRecord-%ED%98%95%EC%8B%9D%EC%9C%BC%EB%A1%9C-%EB%A7%8C%EB%93%9C%EB%8A%94-%EB%B6%88%EB%B3%80Immutable-%EA%B0%9D%EC%B2%B4) 첨부합니다.

Reply:
리뷰해주신대로 ObjectData 를 TableBase 로 옮겨서
public List<TableBase.ObjectData> SeedList; 이렇게 선언하게 되었습니다!

또 알려주신대로 struct에서 record로 변경했습니다.
처음엔 닷넷 버전만 바꾸면 될 줄 알았는데, 해결이 안되어서 더 찾아보니 유니티 자체에서 아직 지원해주지 않는 것 같습니다.
그래서 public string Type { get; **init;** } 형태가 아닌
public string Type { get; } 형태로 접근만 가능하게 하고 초기화는 생성자로 했습니다.
- [C# compiler](https://docs.unity3d.com/2021.3/Documentation/Manual/CSharpCompiler.html)
(You shouldn’t use C# records in serialized types because Unity’s serialization system doesn’t support C# records.)